### PR TITLE
docs: translate « static analysis » from `en` to `fr`

### PR DESCRIPTION
**Pull request by MKlblangenois**

### CODE REVIEW

File: content/post/exodus_static_analysis.fr.md [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-title: "[EN] εxodus static analysis"
+title: "L'analyse statique d'εxodus"
 date: 2018-08-17T22:42:42+02:00
 draft: false
-tags: [analysis, reverse-engineering]
+tags: [analyse, reverse-engineering]
 ---
 
-**Note : ce billet est seulement disponible en anglais pour le moment.**
+L'analyse statique des applications Android est au cœur de la plateforme εxodus. L'idée principale est de détecter la présence de _trackers_ connus dans une application sans l'exécuter. En fait, εxodus ne peut pas prendre trop de temps pour chaque analyse, donc la méthode de détection doit être efficace dans le temps, reproductible et exempte de faux positifs.
```
Comment (Gu1nness) [inline content/post/exodus_static_analysis.fr.md:8]: We use the word «pisteurs» instead of «trackers» in French

**Review by Gu1nness — CHANGES_REQUESTED** [review]

File: content/post/exodus_static_analysis.fr.md [hunk 2]
```diff
@@ -61,16 +62,18 @@ $ dexdump my.apk | grep "Class descriptor" | sort | uniq
   Class descriptor  : 'Landroid/databinding/adapters/AutoCompleteTextViewBindingAdapter$FixText;'
 ```
 
-In Java, C++, Python and other languages, name-spaces are really important and when you provide a library, you have to ensure that the library name-space never collides with other libraries.
+En Java, C++, Python et d'autres langages, les espaces de nom sont vraiment importants et lorsque vous fournissez une bibliothèque, vous devez vous assurer que l'espace de nom de la bibliothèque n'entre jamais en conflit avec d'autres bibliothèques.
+
+εxodus "fouille" dans les fichiers `.apk` pour trouver des _trackers_ et ces _trackers_ peuvent être identifiés par leur propre espace de nom. Par exemple, le `name-space/package` racine de la bibliothèque Amazon Ads est `com.amazon.device.ads`, cette bibliothèque partage le nom du package `com.amazon` avec d'autres bibliothèques Amazon. Ainsi, à ce stade, une signature de suivi est un espace de nom (un nom de _package_).
```
Comment (Gu1nness) [inline content/post/exodus_static_analysis.fr.md:67]: Here as well, «pisteurs» please :)

File: content/post/exodus_static_analysis.fr.md [hunk 3]
```diff
@@ -84,14 +87,15 @@ $ dexdump my.apk | grep "Class descriptor" | sort | uniq | grep -E "com.flurry."
...
+Comme nous pouvons le voir, `my.apk` contient des classes de Flurry.
+
+Ainsi, lorsque vous [soumettez](https://reports.exodus-privacy.eu.org/analysis/submit/) une application pour analyse, εxodus la télécharge depuis Google Play, extrait le fichier `.apk`, appelle `dexdump` sur les fichiers `.dex`, stocke la sortie filtrée dans un fichier qui sera utilisé lors de la mise à jour des rapports afin d'accélérer l'analyse, prend la règle de détection de chaque _tracker_ connu et voit si les noms de classe correspondent. Si une règle de détection correspond à un ou plusieurs noms de classe, nous considérons le _tracker_ intégré, sinon, nous le considérons comme manquant.
```
Comment (Gu1nness) [inline content/post/exodus_static_analysis.fr.md:93]: Here as well, «pisteurs» please :)

File: content/post/exodus_static_analysis.fr.md [hunk 4]
```diff
@@ -84,14 +87,15 @@ $ dexdump my.apk | grep "Class descriptor" | sort | uniq | grep -E "com.flurry."
...
 
 <center>
-{{< fig src="/media/static_analysis.png" caption="Overview of the static analysis process" >}}
+{{< fig src="/media/static_analysis.png" caption="Aperçu du processus d'analyse statique" >}}
 </center>
 
-If you want to help us to improve the εxodus knowledge base, [contact us](/page/who/) and ask for an [ETIP](http://etip.exodus-privacy.eu.org/) account. ETIP is a collaborative platform meant to ease investigation and classification on trackers.
+Si vous souhaitez nous aider à améliorer la base de connaissances εxodus, [contactez-nous](/fr/page/who/) et demandez un compte [ETIP](http://etip.exodus-privacy.eu.org/). ETIP est une plateforme collaborative destinée à faciliter les investigations et la classification sur les trackers.
```
Comment (Gu1nness) [inline content/post/exodus_static_analysis.fr.md:99]: Here as well, «pisteurs» please :)

File: content/post/exodus_static_analysis.fr.md [hunk 5]
```diff
@@ -84,14 +87,15 @@ $ dexdump my.apk | grep "Class descriptor" | sort | uniq | grep -E "com.flurry."
...
 
-If you are an application developer, you can use [exodus-standalone](https://github.com/Exodus-Privacy/exodus-standalone) to analyse your application **before** its publication on Google Play.
+Si vous êtes développeur d'applications, vous pouvez utiliser [exodus-standalone](https://github.com/Exodus-Privacy/exodus-standalone) pour analyser votre application avant sa publication sur Google Play.
```
Comment (Gu1nness) [inline content/post/exodus_static_analysis.fr.md:101]: Please put «avant» surrounded by \*\* :)

**Comment by MKlblangenois** [comment]
@Gu1nness I've updated the article based on your changes request 😊

**Review by Gu1nness — APPROVED** [review]
LGTM

File: content/post/exodus_static_analysis.fr.md [hunk 6]
```diff
@@ -40,9 +39,11 @@ Class #1            -
   Class descriptor  : 'Landroid/databinding/BaseObservable;'
 [...]
 ```
-This output is not really human-friendly but for each class declared in the application we have a `Class #x` and a `Class descriptor`. The class descriptor is the JVM representation of application name-spaces (fully qualified class name). In this example, `Class #0` is `android.databinding.Observable` with `android.databinding` the package name and `Observable` the name of the declared class.
 
-We can cleanup the `dexdump` output by selecting `Class descriptor` and sorting them:
+Cette sortie n'est pas vraiment conviviale mais pour chaque classe déclarée dans l'application, nous avons un `Class # x` et un `Class descriptor`. Le descripteur de classe est la représentation JVM des espaces de nom d'application (nom de classe complet). Dans cet exemple, `Classe # 0` est `android.databinding.Observable` avec `android.databinding` le nom du package et `Observable` le nom de la classe déclarée.
```
Comment (pnu-s) [inline content/post/exodus_static_analysis.fr.md:43]:
```suggestion
Cette sortie n'est pas vraiment lisible mais pour chaque classe déclarée dans l'application, nous avons un `Class # x` et un `Class descriptor`. Le descripteur de classe est la représentation JVM des espaces de nom d'application (nom de classe complet). Dans cet exemple, `Classe # 0` est `android.databinding.Observable` avec `android.databinding` le nom du package et `Observable` le nom de la classe déclarée.
```

File: content/post/exodus_static_analysis.fr.md [hunk 7]
```diff
@@ -1,19 +1,18 @@
...
-title: "[EN] εxodus static analysis"
+title: "L'analyse statique d'εxodus"
 date: 2018-08-17T22:42:42+02:00
 draft: false
-tags: [analysis, reverse-engineering]
+tags: [analyse, reverse-engineering]
 ---
 
-**Note : ce billet est seulement disponible en anglais pour le moment.**
+L'analyse statique des applications Android est au cœur de la plateforme εxodus. L'idée principale est de détecter la présence de pisteurs connus dans une application sans l'exécuter. En fait, εxodus ne peut pas prendre trop de temps pour chaque analyse, donc la méthode de détection doit être efficace dans le temps, reproductible et exempte de faux positifs.
```
Comment (pnu-s) [inline content/post/exodus_static_analysis.fr.md:8]:
```suggestion
L'analyse statique des applications Android est au cœur de la plateforme εxodus. L'idée principale est de détecter la présence de pisteurs connus dans une application sans l'exécuter. En effet, εxodus ne peut pas prendre trop de temps pour chaque analyse, donc la méthode de détection doit être efficace dans le temps, reproductible et exempte de faux positifs.
```

File: content/post/exodus_static_analysis.fr.md [hunk 8]
```diff
@@ -61,16 +62,18 @@ $ dexdump my.apk | grep "Class descriptor" | sort | uniq
   Class descriptor  : 'Landroid/databinding/adapters/AutoCompleteTextViewBindingAdapter$FixText;'
 ```
 
-In Java, C++, Python and other languages, name-spaces are really important and when you provide a library, you have to ensure that the library name-space never collides with other libraries.
+En Java, C++, Python et d'autres langages, les espaces de nom sont vraiment importants et lorsque vous fournissez une bibliothèque, vous devez vous assurer que l'espace de nom de la bibliothèque n'entre jamais en conflit avec d'autres bibliothèques.
+
+εxodus "fouille" dans les fichiers `.apk` pour trouver des pisteurs et ces pisteurs peuvent être identifiés par leur propre espace de nom. Par exemple, le `name-space/package` racine de la bibliothèque Amazon Ads est `com.amazon.device.ads`, cette bibliothèque partage le nom du package `com.amazon` avec d'autres bibliothèques Amazon. Ainsi, à ce stade, une signature de suivi est un espace de nom (un nom de _package_).
```
Comment (pnu-s) [inline content/post/exodus_static_analysis.fr.md:67]:
```suggestion
εxodus "fouille" dans les fichiers `.apk` pour trouver des pisteurs et ces pisteurs peuvent être identifiés par leur propre espace de nom. Par exemple, le `espace-de-nom/packet` racine de la bibliothèque Amazon Ads est `com.amazon.device.ads`, cette bibliothèque partage le nom du package `com.amazon` avec d'autres bibliothèques Amazon. Ainsi, à ce stade, une signature de pisteurs est un espace de nom (un nom de _package_).
```

File: content/post/exodus_static_analysis.fr.md [hunk 9]
```diff
@@ -61,16 +62,18 @@ $ dexdump my.apk | grep "Class descriptor" | sort | uniq
...
 
-εxodus digs into `.apk` files to find trackers and these trackers can be identified by their own name-space. As an example, Amazon Ads library root name-space/package is `com.amazon.device.ads`, this library shares the `com.amazon` package name with other Amazon libraries. So, at this point, a tracker signature is a name-space (a package name).
+Pour le moment, εxodus connaît [152 pisteurs](https://reports.exodus-privacy.eu.org/trackers/), et si nous regardons [Flurry](https://reports.exodus-privacy.eu.org/trackers/25/), nous avons la règle de détection `com.flurry.` qui est un [regex](https://fr.wikipedia.org/wiki/Expression_r%C3%A9guli%C3%A8re).
```
Comment (pnu-s) [inline content/post/exodus_static_analysis.fr.md:69]:
```suggestion
Pour le moment, εxodus connaît [152 pisteurs](https://reports.exodus-privacy.eu.org/trackers/), et si nous regardons [Flurry](https://reports.exodus-privacy.eu.org/trackers/25/), nous avons la règle de détection `com.flurry.` qui est une [regex](https://fr.wikipedia.org/wiki/Expression_r%C3%A9guli%C3%A8re).
```

File: content/post/exodus_static_analysis.fr.md [hunk 10]
```diff
@@ -61,16 +62,18 @@ $ dexdump my.apk | grep "Class descriptor" | sort | uniq
...
 
-By now, εxodus knows [152 trackers](https://reports.exodus-privacy.eu.org/trackers/) and if we have a look to [Flurry](https://reports.exodus-privacy.eu.org/trackers/25/) we have detection rule `com.flurry.` which is a [regex](https://en.wikipedia.org/wiki/Regular_expression).
 <center>
-{{< fig src="/media/flurry.png" caption="Overview of the Flurry description" >}}
+{{< fig src="/media/flurry.png" caption="Aperçu de la description Flurry" >}}
```
Comment (pnu-s) [inline content/post/exodus_static_analysis.fr.md:72]:
```suggestion
{{< fig src="/media/flurry.png" caption="Aperçu de la description du pisteur Flurry" >}}
```

File: content/post/exodus_static_analysis.fr.md [hunk 11]
```diff
@@ -61,16 +62,18 @@ $ dexdump my.apk | grep "Class descriptor" | sort | uniq
...
 </center>
 
-So, lets use `grep` to see if `my.apk` contains Flurry's classes
+Ainsi, utilisons `grep` pour voir si `my.apk` contient les classes de Flurry :
```
Comment (pnu-s) [inline content/post/exodus_static_analysis.fr.md:75]:
```suggestion
Maintenant, utilisons `grep` pour voir si `my.apk` contient les classes de Flurry :
```

File: content/post/exodus_static_analysis.fr.md [hunk 12]
```diff
@@ -84,14 +87,15 @@ $ dexdump my.apk | grep "Class descriptor" | sort | uniq | grep -E "com.flurry."
...
 <center>
-{{< fig src="/media/static_analysis.png" caption="Overview of the static analysis process" >}}
+{{< fig src="/media/static_analysis.png" caption="Aperçu du processus d'analyse statique" >}}
 </center>
 
-If you want to help us to improve the εxodus knowledge base, [contact us](/page/who/) and ask for an [ETIP](http://etip.exodus-privacy.eu.org/) account. ETIP is a collaborative platform meant to ease investigation and classification on trackers.
+Si vous souhaitez nous aider à améliorer la base de connaissances εxodus, [contactez-nous](/fr/page/who/) et demandez un compte [ETIP](http://etip.exodus-privacy.eu.org/). ETIP est une plateforme collaborative destinée à faciliter les investigations et la classification sur les pisteurs.
 
-If you are an application developer, you can use [exodus-standalone](https://github.com/Exodus-Privacy/exodus-standalone) to analyse your application **before** its publication on Google Play.
+Si vous êtes développeur d'applications, vous pouvez utiliser [exodus-standalone](https://github.com/Exodus-Privacy/exodus-standalone) pour analyser votre application **avant** sa publication sur Google Play.
```
Comment (pnu-s) [inline content/post/exodus_static_analysis.fr.md:101]:
```suggestion
Si vous êtes développeur·euse d'applications, vous pouvez utiliser [exodus-standalone](https://github.com/Exodus-Privacy/exodus-standalone) pour analyser votre application **avant** sa publication sur Google Play.
```

**Review by pnu-s — CHANGES_REQUESTED** [review]

**Comment by pnu-s** [comment]
Hi @MKlblangenois and thanks for the proposal!
I finally had the time to review it and I made a couple of suggestions :)

**Review by Gu1nness — APPROVED** [review]

**Comment by pnu-s** [comment]
Thanks again @MKlblangenois :tada:

**Pull request merged**